sys/shell/shell.c

```diff
@@ -271,7 +313,13 @@ static int readline(char *buf, size_t size)
 #endif
         }
         else {
-            *line_buf_ptr++ = c;
+            /* Always consume characters, but do not not always store them */
+            if ((size_t) curr_pos < size - 1) {
```

Comment:
```suggestion
            if ((size_t)curr_pos < size - 1) {
```

Reply:
I've found both variants in the RIOT repository and `CODING_CONVENTIONS.md` doesn't specify how to do this, so I would like to keep it with the space.

Reply:
I applied uncrustify and this was part of the flagged changes. But I'm wont insist on this, thats why I only made a comment and not a change request ;)